src/CodeGen_LLVM.cpp

```diff
@@ -904,6 +904,8 @@ llvm::Type *CodeGen_LLVM::llvm_type_of(Type t) {
     return Internal::llvm_type_of(context, t);
 }
 
+llvm::TargetMachine *get_target_machine(const llvm::Module &module);
```

Comment:
I suggest moving get_target_machine to CodeGen_LLVM.cpp/h (which LLVM_Output.cpp includes) rather than extern declaring it.